<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modellflugvereinsverwaltung Mfvv</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/Airplane_Silhouette_L_64x64.png</normaloff>icons/Airplane_Silhouette_L_64x64.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Modellflugvereinsverwaltung Mfvv </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>488</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_stammdaten">
       <property name="toolTip">
        <string>Stammdaten</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/shiny/64x64/icon-17.png</normaloff>icons/shiny/64x64/icon-17.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Mitglieder">
       <property name="toolTip">
        <string>Mitglieder</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/sketch/PNG - color/64px/handy-icon_08.png</normaloff>icons/sketch/PNG - color/64px/handy-icon_08.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_dokumente">
       <property name="toolTip">
        <string>Dokumente</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/shiny/64x64/icon-18.png</normaloff>icons/shiny/64x64/icon-18.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_termine">
       <property name="toolTip">
        <string>Kalender</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/sketch/PNG - color/64px/handy-icon_04.png</normaloff>icons/sketch/PNG - color/64px/handy-icon_04.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_einstellungen">
       <property name="toolTip">
        <string>Einstellungen</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/sketch/PNG - color/64px/handy-icon_13.png</normaloff>icons/sketch/PNG - color/64px/handy-icon_13.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_info">
       <property name="toolTip">
        <string>Info</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/sketch/PNG - color/64px/handy-icon_01.png</normaloff>icons/sketch/PNG - color/64px/handy-icon_01.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStammdaten">
    <property name="title">
     <string>Stammdaten</string>
    </property>
    <widget class="QMenu" name="menuVereinsdaten">
     <property name="title">
      <string>Vereinsdaten</string>
     </property>
     <addaction name="actionVorstand"/>
     <addaction name="actionBeitrag"/>
     <addaction name="actionBeitragsgruppen"/>
    </widget>
    <addaction name="menuVereinsdaten"/>
    <addaction name="actionUser"/>
    <addaction name="separator"/>
    <addaction name="actionProgramm_beenden"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuMitglieder">
    <property name="title">
     <string>Mitglieder</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="action_ndern"/>
    <addaction name="actionSuchen"/>
    <addaction name="actionListe"/>
    <addaction name="separator"/>
    <addaction name="actionFlugmodelle_anlegen"/>
    <addaction name="actionFlugmodell_zuordnen"/>
    <addaction name="actionL_rmpass_anlegen"/>
    <addaction name="actionL_rmpass_drucken"/>
   </widget>
   <widget class="QMenu" name="menuDokumente">
    <property name="title">
     <string>Dokumente</string>
    </property>
    <addaction name="actionNeu_2"/>
    <addaction name="actionSuchen_2"/>
   </widget>
   <widget class="QMenu" name="menuKalender">
    <property name="title">
     <string>Kalender</string>
    </property>
    <addaction name="actionTermine_anlegen"/>
    <addaction name="actionKalender"/>
    <addaction name="actionVeranstaltungen"/>
   </widget>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="actionBearbeiten"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionInfo_ber"/>
    <addaction name="actionHilfe"/>
   </widget>
   <addaction name="menuStammdaten"/>
   <addaction name="menuMitglieder"/>
   <addaction name="menuDokumente"/>
   <addaction name="menuKalender"/>
   <addaction name="menuEinstellungen"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInfo_ber">
   <property name="text">
    <string>Info Über ...</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
  <action name="actionNeu">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="action_ndern">
   <property name="text">
    <string>Bearbeiten</string>
   </property>
  </action>
  <action name="actionSuchen">
   <property name="text">
    <string>Suchen</string>
   </property>
  </action>
  <action name="actionListe">
   <property name="text">
    <string>Liste</string>
   </property>
  </action>
  <action name="actionFlugmodelle_anlegen">
   <property name="text">
    <string>Flugmodelle anlegen</string>
   </property>
  </action>
  <action name="actionFlugmodell_zuordnen">
   <property name="text">
    <string>Flugmodell zuordnen</string>
   </property>
  </action>
  <action name="actionL_rmpass_anlegen">
   <property name="text">
    <string>Lärmpass anlegen</string>
   </property>
  </action>
  <action name="actionL_rmpass_drucken">
   <property name="text">
    <string>Lärmpass drucken</string>
   </property>
  </action>
  <action name="actionNeu_2">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionSuchen_2">
   <property name="text">
    <string>Suchen</string>
   </property>
  </action>
  <action name="actionBearbeiten">
   <property name="text">
    <string>Bearbeiten</string>
   </property>
  </action>
  <action name="actionTermine_anlegen">
   <property name="text">
    <string>Termine anlegen</string>
   </property>
  </action>
  <action name="actionKalender">
   <property name="text">
    <string>Kalender</string>
   </property>
  </action>
  <action name="actionVeranstaltungen">
   <property name="text">
    <string>Veranstaltungen</string>
   </property>
  </action>
  <action name="actionProgramm_beenden">
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </action>
  <action name="actionVorstand">
   <property name="text">
    <string>Vorstand</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="text">
    <string>User</string>
   </property>
  </action>
  <action name="actionBeitrag">
   <property name="text">
    <string>Beitrag</string>
   </property>
  </action>
  <action name="actionBeitragsgruppen">
   <property name="text">
    <string>Beitragsgruppen</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pb_Mitglieder</tabstop>
  <tabstop>btn_stammdaten</tabstop>
  <tabstop>pb_dokumente</tabstop>
  <tabstop>pb_termine</tabstop>
  <tabstop>pb_info</tabstop>
  <tabstop>pb_einstellungen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
